tests/by-util/test_ls.rs

```diff
@@ -3117,3 +3117,13 @@ fn test_dereference_symlink_file_color() {
         .succeeds()
         .stdout_is(out_exp);
 }
+
+#[test]
+fn test_tab_size_option() {
+    let scene = TestScenario::new(util_name!());
+
+    scene.ucmd().args(&["-T", "3"]).succeeds();
+
```

Comment:
please run rustfmt on this file too

Reply:
Should be done, there was a trailing newline.